pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -0,0 +1,52 @@
+ifdef::context[:parent-context: {context}]
+
+include::../attributes.adoc[]
+
+[id="help"]
+= Help
+
+:context: assembly-pantheon-help
+
+Welcome to Pantheon help.
+
+This document describes the main concepts, requirements and, actions in Pantheon.
+
+include::modules/con_about_pantheon.adoc[leveloffset=+1]
+
+// Section 1
+== Modular documentation
+
+This section describes modular documentation.
+
+include::modules/con_what-is-a-module.adoc[leveloffset=+2]
+
+include::modules/con_comparison-between-modules-and-DITA-topics.adoc[leveloffset=+2]
+
+// Section 2
+include::modules/ref_prerequisites.adoc[leveloffset=+1]
+
+// Section 3
+== Concepts
+
+This section describes the main concepts required to understand and use Pantheon.
```

Comment:
Maybe just make the section heading more descriptive and get rid of this sentence?

== Concepts => == Pantheon Concepts

Reply:
Changed as requested.